lib/api/user_token_service.rb

```diff
@@ -3,21 +3,25 @@ class UserTokenService
     def initialize(config = Settings, args = {})
       @config = config
       @svc_options = args
-      @token_mgr = new_token_mgr(base_config[:module], base_config[:name], api_config)
     end
 
-    attr_accessor :token_mgr
+    def token_mgr(namespace = nil)
+      @token_mgr ||= {}
+      if namespace.nil? || namespace == 'ui' # The default token and UI token share the same TokenStore namespace
+        @token_mgr['ui'] ||= new_token_mgr(base_config[:module], base_config[:name], api_config)
+      elsif namespace == 'ws'
+        @token_mgr['ws'] ||= TokenManager.new('ws', :token_ttl => 5.minutes.to_i)
```

Comment:
Would prefer the 5 minutes is not hardcoded.  Let's define/fetch from Settings.  maybe Settings.websocket.token_ttl ?